src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/multiples/MultiplesSendNotificationController.java

```diff
@@ -0,0 +1,129 @@
+
+    /**
+     * Send Notification about to submit from the Multiple.
+     *
+     * @param multipleRequest holds the request and case data
+     * @param userToken       used for authorization
+     * @return Callback response entity with case data attached.
+     */
+    @PostMapping(value = "/aboutToSubmit", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "aboutToSubmit")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully",
+            content = {
+                @Content(mediaType = "application/json",
+                        schema = @Schema(implementation = MultipleCallbackResponse.class))
+            }),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<MultipleCallbackResponse> aboutToSubmit(
+            @RequestBody MultipleRequest multipleRequest,
+            @RequestHeader("Authorization") String userToken) {
+
+        MultipleDetails caseDetails = multipleRequest.getCaseDetails();
+        MultipleData caseData = caseDetails.getCaseData();
+        List<String> errors = new ArrayList<>();
+        multiplesSendNotificationService.sendNotificationToSingles(
+                caseData,
+                caseDetails,
+                userToken,
+                errors
+        );
+
+        multiplesSendNotificationService.clearSendNotificationFields(caseData);
+        return getMultipleCallbackRespEntity(errors, multipleRequest.getCaseDetails());
+    }
+
+    /**
+     * Returns data needed to populate the submitted page.
+     *
+     * @param multipleRequest holds the request and case data
+     * @return Callback response entity with case data attached.
+     */
+    @PostMapping(value = "/submitted", consumes = APPLICATION_JSON_VALUE)
+    @Operation(summary = "submitted")
+    @ApiResponses(value = {
+        @ApiResponse(responseCode = "200", description = "Accessed successfully",
+            content = {
+                @Content(mediaType = "application/json",
+                        schema = @Schema(implementation = MultipleCallbackResponse.class))
+            }),
+        @ApiResponse(responseCode = "400", description = "Bad Request"),
+        @ApiResponse(responseCode = "500", description = "Internal Server Error")
+    })
+    public ResponseEntity<MultipleCallbackResponse> submitted(@RequestBody MultipleRequest multipleRequest) {
+
+        String caseId = multipleRequest.getCaseDetails().getCaseId();
+        String body = String.format("""
+                ### What happens next
+                The selected parties will receive the notification. </br>
+                The notifications will be stored on the individual case, not on the Multiple. </br>
+                Another notification can be sent <a href="/cases/case-details/%s/trigger/sendNotification/sendNotification1">using this link</a>
```

Comment:
Should this link be to a Multiples endpoint?

Reply:
/trigger makes it start an event rather than go to a controller endpoint. There is an event called sendNotification with case type ET_EnglandWales_Multiple and that's what triggers